styles/_blog.less

```diff
@@ -70,7 +70,7 @@ body:not(.button-style-default) .sqs-editable-button, body:not(.button-style-def
 }
 
 .blog-author-name {
-  color: @badgerRed;
+  color: @badgerRedOnWhite;
```

Comment:
I think I've made a mistake here... this comment was supposed to be made on one of the below changes, so I think this needs reverting, sorry @dstodolny!